docs/source/tutorials/improving_ml_performance.ipynb

```diff
@@ -791,8 +791,8 @@
    "source": [
     "<details><summary>See the implementation of `edit_data` **(click to expand)**</summary>\n",
     "    \n",
-    "    ```python\n",
-    "    # Note: This pulldown content is for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "```python\n",
+    "# Note: This pulldown content is for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "    \n",
     "    def edit_data(train_features, train_labels, label_issues, near_duplicates, outliers, drop_label_issue, drop_near_duplicate, drop_outlier):\n",
```

Comment:
@mturk24 why is this other part all still indented?

And where is your screenshot showing how this fixed the docs build?

Reply:
I can review this, no need for Elias

Reply:
Applied suggested edit, and rebuilt docs locally


<img width="301" alt="Screenshot 2024-07-17 at 10 48 05 PM" src="https://github.com/user-attachments/assets/297082d9-8c17-41c7-8788-cbeb9dab3e92">


When building I had set

`nbsphinx_allow_errors = True`

in conf.py so that's potentially why I still saw the correctly built docs when the python ``` and </details>
were out of order?

Reply:
weird i'm not sure what happened there